setup.py

```diff
@@ -237,32 +236,16 @@ def ctresrc(self):
 _travis_build = os.environ.get("TRAVIS_BUILD")
 libraries = None
 cmdclass = {}
-if ctre_rpi:
-    ext_modules = [
-        Extension(
-            "ctre._impl.ctre_cpp",
-            ["ctre/_impl/ctre_cpp.cpp"],
-            include_dirs=[
-                # Path to pybind11 headers
-                get_pybind_include(),
-                get_pybind_include(user=True),
-                get.ctresrc,
-            ],
-            libraries=libraries,
-            library_dirs=[
-                join(get.ctresrc, "linux", "athena", "static"),
-            ],
-            language="c++",
-        )
-    ]
-    
-# Detect roboRIO.. not foolproof, but good enough
-elif exists("/etc/natinst/share/scs_imagemetadata.ini") or _travis_build:
+# Detect raspberrypi or roboRIO.. not foolproof, but good enough
+if exists("/etc/natinst/share/scs_imagemetadata.ini") or _travis_build or ctre_rpi:
 
     # Don't try to link when testing on travis, as it will fail
     # -> We can still catch compile errors, which is good enough I suspect
-    if not _travis_build:
-        libraries = ["wpiHal", "CTRE_PhoenixCCI"]
+    if not _travis_build: 
+        if ctre_rpi:
+            libraries = ["CTRE_PhoenixCCI"]
+        else:
+            libraries = ["wpiHal", "CTRE_PhoenixCCI"]
```

Comment:
@virtuald does the order of this list matter? If not, I guess we could simplify this by having the common `["CTRE_PhoenixCCI"]` and append `"wpiHal"` for athena.

Reply:
It probably doesn't matter.